pytorch_lightning/trainer/trainer.py

```diff
@@ -635,7 +635,7 @@ def _determine_data_use_amount(self, overfit_batches: float) -> None:
             self.limit_val_batches = 0
 
     def _setup_on_init(self, num_sanity_val_steps: int) -> None:
-        self._log_device_info()
+        # self._log_device_info()
```

Comment:
Why is this commented?

Reply:
I was thinking address #11014 at the same time, haven't get to do it yet. Will uncomment first